en/middleware/mavlink.md

```diff
@@ -105,7 +112,7 @@ protected:
             _msg_ca_trajectory.seq_id = _ca_trajectory.seq_id;
```

Comment:
This line and above define the mapping from uorb to MAVLInk message CA_TRAJECTORY. I think we might help people by providing the definitions of both of those in the doc for people to copy. They could then run the tutorial end to end and know it makes sense. 

For the uorb topic, would this make sense based on the above values? Advice on coefficients?

```
uint64 timestamp				# time since system start (microseconds)
uint64 time_start_usec			# time for trajectory start (microseconds)
uint64 time_stop_usec			# time for trajectory stop (microseconds)
float32 coefficients				# NOT SURE ABOUT THIS - it matches the type, but is singular, so does not match name
uint32 seq_id					# sequence id
```

What do you think of the idea, and if so, is this message OK or we could just pretend that BATTERY_STATUS say is a new message?

@julianoes Opinions?

Reply:
I think using an existing one would be a bit simpler. Then we can also directly point to the messages.

Reply:
Thank you!